src/java/com/wolfssl/provider/jsse/WolfSSLImplementSSLSession.java

```diff
@@ -147,12 +147,13 @@ public WolfSSLImplementSSLSession (WolfSSLSession in, int port, String host,
      *
      * @param in WolfSSLSession to be used with this object
      * @param params WolfSSLAuthStore for this session
+     * @param host hostname of peer, or null if not available
+     * @param port port of peer
      */
     public WolfSSLImplementSSLSession (WolfSSLSession in,
-                                       WolfSSLAuthStore params) {
+                                       WolfSSLAuthStore params,
+                                       String host, int port) {
```

Comment:
Are these exposed publicly? Would a user being calling the previous versions?

Reply:
Technically this is a public constructor, but realistically the only person/area calling this is our own JSSE layer.  This should be a safe change.